test/AutoRest.TestServer.Tests/Mgmt/Unit/CodeModelExtensionTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using AutoRest.CSharp.Mgmt.Decorator;
+using NUnit.Framework;
+
+namespace AutoRest.TestServer.Tests.Mgmt.Unit
```

Comment:
Can we also add a test class that inherits from testproject tests and asserts the specific expected names that we want to change in the mgmtrenamerules project?

Reply:
I will add that